quickjs/src/main/jni/Context.cpp

```diff
@@ -471,7 +471,7 @@ Context::getJsToJavaConverter(JNIEnv* env, jclass type, bool boxed) {
             for (int i = 0; i < length && !env->ExceptionCheck(); i++) {
               int element;
               auto jsElement = JS_GetPropertyUint32(c->jsContext, v, i);
-              if (!JS_IsInteger(jsElement)) {
+              if (JS_VALUE_GET_TAG(jsElement) != JS_TAG_INT) {
```

Comment:
This function was removed. It previously checked for values which were either `JS_TAG_INT` or `JS_TAG_BIG_INT`. I've changed it to check only for `JS_TAG_INT` because calling `length` on an array calls `JS_NewInt32` which creates a value of this specific type.